Frameworks/CoreLocation/CLHeadingInternal.h

```diff
@@ -0,0 +1,26 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+#pragma once
+
+#import <CoreLocation/CoreLocationExport.h>
+#import <Foundation/NSObject.h>
+
+@interface CLHeading ()
+- (instancetype)initWithAccuracy:(CLLocationDirection)accuracy
+                 magneticHeading:(CLLocationDirection)magneticHeading
```

Comment:
Apparently it is. That's a very unusual API design on the part of the reference platform.

---
In reply to: [70519373](https://github.com/Microsoft/WinObjC/pull/635#discussion_r70519373) [](ancestors = 70519373)